### DISCUSSION TITLE
[灌水]炫酷的绳匠ID征集

### DISCUSSION THREAD

**Discussion by Flshey** [share121/inter-knot#883] [其他]
我好后悔啊，当初只是抱着看戏吃瓜的目的来这里。嫌麻烦就随便瞎起了个名字。现在想开始从事绳匠工作了，这名字…我…唉!只能重新注册一个新号了，换号之前征集波炫酷的绳匠名字!

**Comment by sky27950** [comment]
无敌暴龙战神(*^_^*)

**Comment by ShuoChen08** [comment]
fLshey

**Comment by PaiweiBilibili** [comment]
赞同一楼哈哈哈

**Comment by capacitor1** [comment]
炫酷的ID？这里有一个网站专门在收集这种ID，建议[点击链接](https://github.com/share121/inter-knot/discussions/883)前往查看。

**Comment by match-fire** [comment]
FIshey，带着这个名字一定能成为传奇绳匠
